ocaml/xapi/xapi_vtpm.ml

```diff
@@ -45,11 +45,16 @@ let destroy ~__context ~self =
 
 let get_contents ~__context ~self =
   let secret = Db.VTPM.get_contents ~__context ~self in
-  Base64.decode_exn (Db.Secret.get_value ~__context ~self:secret)
+  Db.Secret.get_value ~__context ~self:secret
```

Comment:
This is a breaking change for anyone who has had VM's saved with the previous encoding, but that is probably just me at the moment, and I can probably manually update the DB following this change (otherwise we'd need  DB upgrade rule, but probably not worth writing one now)